bin/tink.js

```diff
@@ -4,12 +4,13 @@ require('../lib/node/index.js')
 
 const CMDS = new Map([
   ['add', require('../lib/commands/add.js')],
-  ['rm', require('../lib/commands/rm.js')],
-  ['shell', require('../lib/commands/shell.js')],
+  ['build', require('../lib/commands/build.js')],
+  ['deprecate', require('../lib/commands/deprecate.js')],
   ['org', require('../lib/commands/org.jsx')],
   ['prepare', require('../lib/commands/prepare.js')],
   ['ping', require('../lib/commands/ping.js')],
-  ['deprecate', require('../lib/commands/deprecate.js')]
+  ['rm', require('../lib/commands/rm.js')],
+  ['shell', require('../lib/commands/shell.js')]
```

Comment:
Minor FYI: ☝️while adding the required `build.js`, why not sort these items alphabetically.
(Happy to revert sort if it's bad idea.)